jmh-core/src/main/java/org/openjdk/jmh/profile/MemPoolProfiler.java

```diff
@@ -67,6 +70,16 @@ public Collection<? extends Result> afterIteration(BenchmarkParams benchmarkPara
 
         results.add(new ScalarResult("mempool.total.codeheap.used", sumCodeHeap / BYTES_PER_KIB, "KiB", AggregationPolicy.MAX));
         results.add(new ScalarResult("mempool.total.used", sum / BYTES_PER_KIB, "KiB", AggregationPolicy.MAX));
+
+        results.addAll(
```

Comment:
These need to be added to `mempool.total.used`. Also, rewrite it in the same style as existing code, without streams.

Reply:
Thanks, done